components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/connector/RecoveryConfigImpl.java

```diff
@@ -89,6 +91,9 @@ public int getOrder() {
                 "Set false if the client application handles notification sending");
         descriptionMapping.put(IdentityRecoveryConstants.ConnectorConfig.RECOVERY_QUESTION_PASSWORD_RECAPTCHA_ENABLE,
                 "Show captcha for challenge question based password recovery");
+        descriptionMapping.put(IdentityRecoveryConstants.ConnectorConfig.FORCE_ADD_PW_RECOVERY_QUESTION,
+                "This setting will force to get answers to challenge questions while the user logs in if not" +
```

Comment:
shall we re-word as "Force users to provide answers to challenge questions during log-in"